appinventor/appengine/src/com/google/appinventor/server/storage/ObjectifyStorageIo.java

```diff
@@ -222,7 +230,11 @@ public IOException getIOException() {
       }
       LOG.log(Level.INFO, "Default GCS Bucket Configured from App Identity: " + gcsBucket);
     }
+    if (apkBucket.equals("")) { // Default to same as gcsBucket
+      apkBucket = gcsBucket;
+    }
     GCS_BUCKET_NAME = gcsBucket;
+    APK_BUCKET_NAME = apkBucket;
```

Comment:
If the previous optimization is taken, then `apkBucket` becomes redundant and calling `Flag.get()` could be provided here.